packages/solid-router/src/HeadContent.tsx

```diff
@@ -130,7 +131,8 @@ export const useTags = () => {
  */
 export function HeadContent() {
   const tags = useTags()
-  return tags.map((tag) => <Asset {...tag} />)
+  useAssets(() => tags.map((tag) => <Asset {...tag} />))
+  return null
```

Comment:
do we need to change this?

Reply:
Yeah otherwise it just renders the assets as string